pkg/apis/apis/v1alpha1/types.go

```diff
@@ -140,6 +147,19 @@ type APIBindingStatus struct {
 	//
 	// +optional
 	Conditions conditionsv1alpha1.Conditions `json:"conditions,omitempty"`
+
+	// observedAcceptedPermissionClaims records the permissions that the export provider is granted
+	// to the bound workspace. This is granted by binding implictily to a export that contains
+	// permissionClaims.
+	// Access is granted on a GroupResource basis and can be filtered on objects by many different selectors.
+	// +optional
+	ObservedAcceptedPermissionClaims []PermissionClaim `json:"ObservedAcceptedPermissionClaims,omitempty"`
+}
+
+type PermissionClaimLabel struct {
+	GroupResource `json:","`
+	Key           string
```

Comment:
needs json annotations?

Reply:
needed to remove all of this. good catch.